cli/cli.go

```diff
@@ -300,6 +301,20 @@ func initialModel(prompt string, client *openai.OpenAIClient) model {
 
 // === Main === //
 
+func getShellSyntax() string {
+	if runtime.GOOS == "windows" {
+		return "\n```powershell\n$env:OPENAI_API_KEY = \"[your key]\"\n```"
+	}
+	return "\n```bash\nexport OPENAI_API_KEY=[your key]\n```"
+}
+
+func getProfileScriptName() string {
+	if runtime.GOOS == "windows" {
+		return "$profile"
+	}
+	return ".zshrc or .bashrc"
+}
+
```

Comment:
Maybe let's make these variables inside `printAPIKeyNotSetMessage()` instead of functions. Kind of like:

```go
set_api_key_syntax := [default to linux]
profile_file_syntax := [default to zsh and bash]
if runtime.GOOS == "windows" {
  [set them to windows values]
}
```